cmd/rm-main.go

```diff
@@ -88,30 +88,33 @@ ENVIRONMENT VARIABLES:
 
 EXAMPLES:
    1. Remove a file.
-      $ {{.HelpName}} 1999/old-backup.tgz
+	  $ {{.HelpName}} 1999/old-backup.tgz
+	  
+   2. Perform a fake remove operation.
+	  $ {{.HelpName}} --fake 1999/old-backup.tgz	  
 
-   2. Remove all objects recursively from bucket 'jazz-songs' matching 'louis' prefix.
-      $ {{.HelpName}} --recursive s3/jazz-songs/louis/
+   3. Remove all objects recursively from bucket 'jazz-songs' matching 'louis' prefix.
+      $ {{.HelpName}} --recursive --force s3/jazz-songs/louis/
 
-   3. Remove all objects older than '90' days recursively from bucket 'jazz-songs' that match 'louis' prefix.
-      $ {{.HelpName}} --recursive --older-than 90d s3/jazz-songs/louis/
+   4. Remove all objects older than '90' days recursively from bucket 'jazz-songs' that match 'louis' prefix.
+      $ {{.HelpName}} --recursive --force --older-than 90d s3/jazz-songs/louis/
 
-   4. Remove all objects newer than 7 days and 10 hours recursively from bucket 'pop-songs'
-      $ {{.HelpName}} --recursive --newer-than 7d10h s3/pop-songs/
+   5. Remove all objects newer than 7 days and 10 hours recursively from bucket 'pop-songs'
+      $ {{.HelpName}} --recursive --force --newer-than 7d10h s3/pop-songs/
 
-   5. Remove all objects read from STDIN.
+   6. Remove all objects read from STDIN.
       $ {{.HelpName}} --force --stdin
 
-   6. Remove all objects recursively from S3 host
-      $ {{.HelpName}} --recursive --dangerous s3
+   7. Remove all objects recursively from S3 host
```

Comment:
To be consistent in how we refer to S3 alias, we could say this like, 
```suggestion
   7. Remove all objects recursively from Amazon S3 cloud storage.
```

Reply:
Modified.